spec/features/user_signs_up_spec.rb

```diff
@@ -25,11 +28,84 @@
     expect(page).to have_content('Sign Out')
   end
 
-  scenario 'provide invalid registration information' do
+  scenario 'provide valid registration information - with avatar' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.